code/modules/projectiles/guns/projectile/battle_rifle/levergun.dm

```diff
@@ -0,0 +1,75 @@
+/obj/item/gun/projectile/boltgun/levergun
+	name = "FS BR .40 \"Svengali\""
+	desc = "A well-made Frozen Star battle rifle. Uses .40 rounds."
+	icon_state = "lever_winchester"
+	item_suffix  = "_winchester"
+	force = WEAPON_FORCE_DANGEROUS
+	armor_penetration = ARMOR_PEN_GRAZING
+	caliber = CAL_MAGNUM
+	damage_multiplier = 1.6
```

Comment:
I kinda think these values are extreme, but I support Humon's point on this - we could make the rifle non-marshal only

Reply:
Values are tested and working correctly, they are alright and considering we don't have sprites for this guns gun parts I will have to disagree.